test/acceptance/behave/components/ngsiv2/types/retrieve_entity_types.feature

```diff
@@ -0,0 +1,797 @@
+  I would like to get entity types using NGSI v2 API
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  Actions After each Scenario:
+  Setup: delete database in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+  @happy_path @BUG_1636 @skip
+  Scenario:  get entity types using NGSI v2 API
+    Given  a definition of headers
+      | parameter          | value                |
+      | Fiware-Service     | test_type_happy_path |
+      | Fiware-ServicePath | /test                |
+      | Content-Type       | application/json     |
+    And initialize the accumulator context of entities
+    And create "5" entities with "2" attributes
+      | parameter        | value       |
+      | entities_type    | home        |
+      | entities_id      | room1       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    And verify that receive several "Created" http code
+    And accumulate context of entities for use with lists
+    And create "5" entities with "2" attributes
+      | parameter        | value                   |
+      | entities_id      | room2                   |
+      | attributes_name  | timestamp               |
+      | attributes_value | 017-06-17T07:21:24.238Z |
+      | attributes_type  | date                    |
+      | metadatas_number | 2                       |
+      | metadatas_name   | very_hot                |
+      | metadatas_type   | alarm                   |
+      | metadatas_value  | random=10               |
+    And verify that receive several "Created" http code
+    And accumulate context of entities for use with lists
+    And create "5" entities with "2" attributes
+      | parameter        | value     |
+      | entities_type    | car       |
+      | entities_id      | vehicle   |
+      | attributes_name  | brake     |
+      | attributes_value | 25        |
+      | attributes_type  | seconds   |
+      | metadatas_number | 2         |
+      | metadatas_name   | very_hot  |
+      | metadatas_type   | alarm     |
+      | metadatas_value  | random=10 |
+    And verify that receive several "Created" http code
+    And accumulate context of entities for use with lists
+    And create "5" entities with "2" attributes
+      | parameter        | value |
+      | entities_type    | car   |
+      | entities_id      | moto  |
+      | attributes_name  | speed |
+      | attributes_value | 45    |
+    And verify that receive several "Created" http code
+    And accumulate context of entities for use with lists
+    When get entity types
+    Then verify that receive an "OK" http code
+    And verify that entity types are returned in response
```

Comment:
Which types? I understand that they should be included explictely (e.g. in a table)... otherwise, the test case script is "opaque".